src/js/i18n/en-US.yml

```diff
@@ -425,7 +425,7 @@ careOptsFrontend:
         countView:
           actionsCount: "{itemCount, plural, one {# Action} other {# Actions}}"
           flowsCount: "{itemCount, plural, one {# Flow} other {# Flows}}"
-          maximumListCount: Showing {maximumCount} of many {isFlowList, select, true {Flows} false {Actions}}.
+          maximumListCount: Showing {maximumCount} of {totalInDb} {isFlowList, select, true {Flows} false {Actions}}.
```

Comment:
Had another thought.. 
seems like we can remove `MAXIMUM_LIST_COUNT`. This message just shows up if the `count < total` and then the narrowed version only shows if `count < total && filteredCount < count` correct?